### ISSUE TITLE
segfault if output dir has directory with name of simulation case

### ISSUE THREAD

**Issue by hnil** [OPM/opm-simulators#1672]
Example
```sh
flow --output-dir=test SPE1CASE1.DATA
```
where test/SPE1CASE1 exist

output:
```
**********************************************************************
*                                                                    *
*                      This is flow 2018.10-pre                      *
*                                                                    *
* Flow is a simulator for fully implicit three-phase black-oil flow, *
*             including solvent and polymer capabilities.            *
*          For more information, see https://opm-project.org          *
*                                                                    *
**********************************************************************

Segmentation fault (core dumped)
```
with gdb:
bt (at error)
```
#0  __strlen_avx2 () at ../sysdeps/x86_64/multiarch/strlen-avx2.S:62
#1  0x00007ffff6ade4a9 in util_alloc_filename (path=0x5555583f9d70 "test/SPE1CASE1", basename=0x0, extension=extension@entry=0x0) at /data/hnil/GITHUB/OPM/opm_source/master/libecl/lib/util/util.c:4812
#2  0x00007ffff6b0fdbe in ecl_sum_set_case (ecl_sum=ecl_sum@entry=0x5555583c6070, input_arg=input_arg@entry=0x7fffffffbbf0 "test/SPE1CASE1")
    at /data/hnil/GITHUB/OPM/opm_source/master/libecl/lib/ecl/ecl_sum.cpp:138
#3  0x00007ffff6b0feab in ecl_sum_alloc__ (input_arg=0x7fffffffbbf0 "test/SPE1CASE1", key_join_string=key_join_string@entry=0x5555579d6c46 ":")
    at /data/hnil/GITHUB/OPM/opm_source/master/libecl/lib/ecl/ecl_sum.cpp:166
#4  0x00007ffff6b0ff2b in ecl_sum_alloc_writer__ (ecl_case=<optimized out>, restart_case=0x0, restart_step=-1, fmt_output=<optimized out>, unified=<optimized out>, key_join_string=0x5555579d6c46 ":", 
    sim_start=1420070400, time_in_days=true, nx=10, ny=10, nz=3) at /data/hnil/GITHUB/OPM/opm_source/master/libecl/lib/ecl/ecl_sum.cpp:333
#5  0x0000555557557073 in Opm::out::Summary::Summary (this=0x5555583f9c70, st=..., sum=..., grid_arg=..., schedule=..., basename=0x7fffffffbbf0 "test/SPE1CASE1")
    at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/opm-common/src/opm/output/eclipse/Summary.cpp:956
#6  0x0000555557556cd5 in Opm::out::Summary::Summary (this=0x5555583f9c70, st=..., sum=..., grid_arg=..., schedule=...)
    at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/opm-common/src/opm/output/eclipse/Summary.cpp:924
#7  0x00005555575484ab in Opm::EclipseIO::Impl::Impl (this=0x5555583f9b80, eclipseState=..., grid_=..., schedule_=..., summary_config=...)
    at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/opm-common/src/opm/output/eclipse/EclipseIO.cpp:230
#8  0x000055555754a19e in Opm::EclipseIO::EclipseIO (this=0x5555583c5eb0, es=..., grid=..., schedule=..., summary_config=...)
    at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/opm-common/src/opm/output/eclipse/EclipseIO.cpp:516
#9  0x00005555567d365c in Ewoms::EclWriter<Ewoms::Properties::TTag::EclFlowProblem>::EclWriter (this=0x5555583dd880, simulator=...)
    at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/ewoms/ebos/eclwriter.hh:115
#10 0x000055555679d96b in Ewoms::EclProblem<Ewoms::Properties::TTag::EclFlowProblem>::EclProblem (this=0x5555583ed0a0, simulator=...)
    at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/ewoms/ebos/eclproblem.hh:493
#11 0x000055555678a588 in Ewoms::Simulator<Ewoms::Properties::TTag::EclFlowProblem>::Simulator (this=0x55555826f3a0, verbose=false)
    at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/ewoms/ewoms/common/simulator.hh:134
#12 0x0000555556775149 in Opm::FlowMainEbos<Ewoms::Properties::TTag::EclFlowProblem>::setupEbosSimulator (this=0x7fffffffc4d0)
    at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/opm-simulators/opm/autodiff/FlowMainEbos.hpp:443
#13 0x0000555556769c28 in Opm::FlowMainEbos<Ewoms::Properties::TTag::EclFlowProblem>::execute (this=0x7fffffffc4d0, argc=3, argv=0x7fffffffc9f8)
    at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/opm-simulators/opm/autodiff/FlowMainEbos.hpp:217
#14 0x0000555556742699 in Opm::flowEbosBlackoilMain (argc=3, argv=0x7fffffffc9f8) at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/opm-simulators/opm/simulators/flow_ebos_blackoil.cpp:59
#15 0x0000555556634338 in main (argc=3, argv=0x7fffffffc9f8) at /home/hnil/Documents/GITHUB/OPM/opm_source/devel_adjoint/opm-simulators/examples/flow.cpp:216
```

**Comment by joakim-hove** [comment]
This wax fixed - last minute before the 2018.10 release; so upgrading either to the 2018.10 release or building yourself from source, should fix this.

**Comment by GitPaean** [comment]
closing this, feel free to reopen it, @hnil
